exporter/clickhouselogsexporter/migrations/000001_init_db.up.sql

```diff
@@ -0,0 +1,60 @@
+-- https://altinity.com/blog/2019/7/new-encodings-to-improve-clickhouse
+CREATE TABLE IF NOT EXISTS logs (
+	timestamp UInt64 CODEC(DoubleDelta, LZ4),
+	observed_timestamp UInt64 CODEC(DoubleDelta, LZ4),
+	id String CODEC(ZSTD(1)),
+	trace_id String CODEC(ZSTD(1)),
+	span_id String CODEC(ZSTD(1)),
+	trace_flags UInt32,
+	severity_text LowCardinality(String) CODEC(ZSTD(1)),
+	severity_number UInt8,
+	body String CODEC(ZSTD(2)),
+	resources_string_key Array(String) CODEC(ZSTD(1)),
+	resources_string_value Array(String) CODEC(ZSTD(1)),
+	attributes_string_key Array(String) CODEC(ZSTD(1)),
+	attributes_string_value Array(String) CODEC(ZSTD(1)),
+	attributes_int64_key Array(String) CODEC(ZSTD(1)),
+	attributes_int64_value Array(Int64) CODEC(ZSTD(1)),
+	attributes_float64_key Array(String) CODEC(ZSTD(1)),
+	attributes_float64_value Array(Float64) CODEC(ZSTD(1)),
+	INDEX body_idx body TYPE tokenbf_v1(10240, 3, 0) GRANULARITY 4
+) ENGINE MergeTree()
+PARTITION BY toDate(timestamp / 1000000000)
+ORDER BY (timestamp, id);
+
+
+CREATE TABLE IF NOT EXISTS logs_atrribute_keys (
+name String,
+datatype String
+)ENGINE = ReplacingMergeTree
+ORDER BY (name);
```

Comment:
Should this also include the `datatype` here? Otherwise merging will be just based on `name` which can be incorrect.

Reply:
so you cannot parse two attributes with the same name as attributes is a map in the spec.

Reply:
Name can be single but it's type can change?

Reply:
got the point